### DISCUSSION TITLE
A couple of requests from Japan

### DISCUSSION THREAD

**Discussion by Hiro-Tokyo** [code4recovery/12-step-meeting-list#1151] [Ideas]
A couple of requests from Japan. Just requests from us but If the team could consider for them, we would very appreciate.

1. Adopt color, bold, highlight texts for meeting notes.
2. In Japan, some groups hold 'not every week' meetings, for example, just hold on 1st Monday / not hold on 5th Monday so if TSLM UI calendar format could be arranged, it would be helpful.

If you have any questions, please let us know!

**Comment by kiyote33** [comment]
Hi @Hiro-Tokyo,
We get both your suggestions fairly often (see #948 and #1068 for responses from @joshreisner).  Hope this helps...

**Reply by Hiro-Tokyo** [reply]
Thank you @kiyote33 for your answer! I understand and I will tell about it to Japanese fellowship!
